quiz_app/permissions.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import permissions
+
+
+class IsStaffOrReadOnly(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        return bool(
+            request.method in permissions.SAFE_METHODS or
+            request.user and
+            request.user.is_staff
+        )
+
+
+class IsAuthenticatedOrReadOnly(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+
+        return request.user and request.user.is_authenticated
+
+
+class UpdateOwnFeedback(permissions.BasePermission):
```

Comment:
Naming should be consistent with permission classes. Check naming of other permission classes.